metadata/v20/v20/build_source/scripts/example_fields.json

```diff
@@ -0,0 +1,127 @@
+        "nullable": false,
+        "unit": null,
+        "isAbout": [
+            {
+                "name": null,
+                "path": null
+            }
+        ],
+        "valueReference": [
+            {
+                "value": null,
+                "name": null,
+                "path": null
+            }
+        ]
+    },
+    {
+        "name": "name",
+        "description": "Example name",
+        "type": "text",
+        "nullable": true,
+        "unit": null,
+        "isAbout": [
+            {
+                "name": "written name",
+                "path": "https://openenergy-platform.org/ontology/oeo/IAO_0000590"
+            }
+        ],
+        "valueReference": [
+            {
+                "value": null,
+                "name": null,
+                "path": null
+            }
+        ]
+    },
+    {
+        "name": "type",
+        "description": "Type of wind farm",
+        "type": "text",
+        "nullable": true,
+        "unit": null,
+        "isAbout": [
+            {
+                "name": "wind farm",
+                "path": "https://openenergy-platform.org/ontology/oeo/OEO_00000447"
+            }
+        ],
+        "valueReference": [
+            {
+                "value": "onshore ",
+                "name": "onshore wind farm",
+                "path": "https://openenergy-platform.org/ontology/oeo/OEO_00000311"
+            },
+            {
+                "value": "offshore ",
+                "name": "offshore wind farm",
+                "path": "https://openenergy-platform.org/ontology/oeo/OEO_00000308"
+            }
+        ]
+    },
+    {
+        "name": "year",
+        "description": "Reference year",
+        "type": "integer",
+        "nullable": true,
+        "unit": null,
+        "isAbout": [
+            {
+                "name": "year",
+                "path": "https://openenergy-platform.org/ontology/oeo/UO_0000036"
+            }
+        ],
+        "valueReference": [
+            {
+                "value": null,
+                "name": null,
+                "path": null
+            }
+        ]
+    },
+    {
+        "name": "value",
+        "description": "Example value",
+        "type": "double precision",
+        "nullable": true,
+        "unit": "MW",
+        "isAbout": [
+            {
+                "name": "quantity value",
+                "path": "https://openenergy-platform.org/ontology/oeo/OEO_00000350"
+            }
+        ],
+        "valueReference": [
+            {
+                "value": null,
+                "name": null,
+                "path": null
+            }
+        ]
+    },
+    {
+        "name": "geom",
+        "description": "Geometry",
+        "type": "geometry(Point, 4326)",
+        "nullable": true,
+        "unit": null,
+        "isAbout": [
+            {
+                "name": "spatial region",
+                "path": "https://openenergy-platform.org/ontology/oeo/BFO_0000006"
```

Comment:
the "path" in these fields should be "@id". This reflects the latest updates i think.